### DISCUSSION TITLE
Firebase를 대체할 오픈소스 프로젝트, Supabase - peterkimzz

### DISCUSSION THREAD

**Discussion by giscus** [peterkimzz/blog#17] [Blog Comments]
# Firebase를 대체할 오픈소스 프로젝트, Supabase - peterkimzz

Supabase는 구글 Firebase를 엔터프라이즈 레벨에서도 사용 가능하도록 만든 오픈소스 프로젝트이다. 현재는 베타 서비스이다.
컴퓨터에 직접 설치하는 방식은 아니고, Firebase처럼 클라우드로 제공되는 서비스이다.
내가 Firebase를 사용하면서 아쉬웠던 

https://www.peterkimzz.com/supasbase-overview/

**Comment by hmmhmmhm** [comment]
안녕하세요 국내 Supabase 커뮤니티를 만들고 있습니다!
Supabase 공부하거나 사용하시는 분들 참여 부탁드립니다!
https://open.kakao.com/o/gyq8TWjf
